### PR TITLE
fix spelling errors in docs

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR addresses spelling errors in the docs raised by executing `mdspell -n -a --en-gb '**/*.md' '!docker-images/*.md' '!docs/api/**/*.md'
` in root (https://www.npmjs.com/package/markdown-spellcheck)

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: docs/.spelling [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+faucet
+Fetch.ai.
+unregisters
+teardown
+unregister
+seller_AEA
+buyer_AEA
+Agentland
+generic_seller
+instantiable
```
Comment (marcofavorito) [inline docs/.spelling:63]: isn't a legal word?

File: docs/.spelling [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+startup
```
Comment (marcofavorito) [inline docs/.spelling:64]: maybe `start-up` gets accepted? I don't think so.

File: docs/.spelling [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+backend
```
Comment (marcofavorito) [inline docs/.spelling:65]: try `back-end` or `back end`.
The last one seems preferable, although less widespread.
https://en.wikipedia.org/wiki/Front_end_and_back_end

File: docs/.spelling [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+SQLite
+backends
```
Comment (marcofavorito) [inline docs/.spelling:67]: see above

File: docs/.spelling [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+permissionless
+libp2p
+ORMs
+standalone
+Multiaddresses
+namespaced
+prometheus
+Grafana
+toolset
+deserialised
```
Comment (marcofavorito) [inline docs/.spelling:105]: British vs American spelling?

**Review by marcofavorito — APPROVED** [review]
Very useful. I wonder if we could reduce the number of white-listed words by playing with hyphens or spaces in composite words, different types of spelling etc.

File: docs/.spelling (hunk 1 above)
Comment (DavidMinarsch) [reply docs/.spelling:63]: https://en.wiktionary.org/wiki/instantiable

File: docs/.spelling (hunk 2 above)
Comment (DavidMinarsch) [reply docs/.spelling:64]: its a valid noun, search `define: startup`

File: docs/.spelling (hunk 3 above)
Comment (DavidMinarsch) [reply docs/.spelling:65]: its a valid noun, search `define: startup`

File: docs/.spelling (hunk 4 above)
Comment (DavidMinarsch) [reply docs/.spelling:67]: see above

File: docs/.spelling (hunk 5 above)
Comment (DavidMinarsch) [reply docs/.spelling:105]: Everything is set for British

**Comment by DavidMinarsch** [comment]
> Very useful. I wonder if we could reduce the number of white-listed words by playing with hyphens or spaces in composite words, different types of spelling etc.

That only makes sense when the resulting words are more common. I think the white-listed words are valid additions (checked each one of them).

File: docs/.spelling (hunk 2 above)
Comment (marcofavorito) [reply docs/.spelling:64]: Yes, this and other comments were to reduce at the minimum the size of `.spelling`. But as you said, it's OK.

**Pull request merged**